### DISCUSSION TITLE
v1.0.0-M36

### DISCUSSION THREAD

**Discussion by armanbilge** [http4s/http4s#6637] [Releases]
This release is the latest milestone in the 1.x series, and is _not_ binary compatible with previous 1.0 milestones.  It also includes the changes from v0.23.15.

### http4s-core

* Use `scodec.bits.ByteVector` for `Entity.Strict` by @armanbilge in https://github.com/http4s/http4s/pull/6528
* Smite `Seq` in `Message#multiParams`, `Uri#multiParams`, `Query#multiParams` by @danicheg in https://github.com/http4s/http4s/pull/6524

### http4s-server

* Spring cleaning: Remove unneeded temporal constraints and make some cosmetic changes by @FrancescoSerra in https://github.com/http4s/http4s/pull/6491

### Documentation

* Remove references to Scala 2.12 in docs by @danicheg in https://github.com/http4s/http4s/pull/6604
* Fix 1.0 version table compatibility with 2.12 & 2.13 by @JackTreble in https://github.com/http4s/http4s/pull/6607
* Tweak the http4s.g8 example on the quick start page by @danicheg in https://github.com/http4s/http4s/pull/6608

### Behind the scenes

* Release v1.0.0-M35 by @armanbilge in https://github.com/http4s/http4s/pull/6569
* Pick Sbt Scoverage from 0.23 by @danicheg in https://github.com/http4s/http4s/pull/6574
* Ignore literally updates on main by @armanbilge in https://github.com/http4s/http4s/pull/6584
* Merge `series/0.23` into `main` by @danicheg in https://github.com/http4s/http4s/pull/6612
* 0.23 -> main by @armanbilge in https://github.com/http4s/http4s/pull/6631

## New Contributors

* @JackTreble made their first contribution in https://github.com/http4s/http4s/pull/6607

**Full Changelog**: https://github.com/http4s/http4s/compare/v1.0.0-M35...v1.0.0-M36

<hr /><em>This discussion was created from the release <a href='https://github.com/http4s/http4s/releases/tag/v1.0.0-M36'>v1.0.0-M36</a>.</em>

**Comment by armanbilge** [comment]
Tracking releases of the ancillary repos.  Modules where a volunteer maintainer steps up cut to the front of the line! :smile:

* [ ] armeria
* [x] blaze https://github.com/http4s/blaze/releases/tag/v1.0.0-M36
* [ ] boopickle
* [x] dom https://github.com/http4s/http4s-dom/releases/tag/v1.0.0-M36
* [ ] fabric
* [ ] feral
* [ ] finagle
* [x] jdk-http-client https://github.com/http4s/http4s-jdk-http-client/releases/tag/v1.0.0-M5
* [ ] jetty
* [ ] netty
* [ ] okhttp-client
* [ ] play-json
* [ ] prometheus-client
* [ ] rho
* [ ] scala-xml
* [ ] scalatags
* [ ] servlet
* [ ] session
* [ ] tomcat
* [ ] twirl

**Comment by wjoel** [comment]
@armanbilge is there a description of what one would have to do, as a prospective volunteer maintainer, in order to help out with those ancillary repos? Perhaps I could help out with blaze (and maybe others, if it's simple routine work, but I don't use any of them at the moment).

**Reply by armanbilge** [reply]
@wjoel thanks for volunteering! Essentially most of the work is:

1. updating to the latest http4s milestone every time a new one is released (usually just merging a steward PR, but perhaps addressing some deprecations or other breaking changes)
2. and then releasing a corresponding milestone of that ancillary repo.

So pretty routine. Of course, help on any of the repos is always appreciated, but if you are not using them (and don't plan to be) it's not strictly necessary.

One of the goals of the schism is to find out which ancillary repos are actually used and have users that care enough about them to come out of the woodwork and keep them alive. If a repo is unmaintained and nobody is complaining ... well that says something :)
